### ISSUE TITLE
Add unzipped .cia to releases (for future releases)

### ISSUE THREAD

**Issue by thesouldemon** [J-D-K/JKSM#11]
It would be beneficial to many people(and bots) if you add the .cia in addition to the .zip in the github releases.

**Comment by J-D-K** [comment]
Yes, I know. The reason it's zipped is because I didn't have anything in place to download the filter text file then. Without it, all dev apps and homebrew show up in the SD titles list.
